jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/impl/Bpmn2JsonUnmarshaller.java

```diff
@@ -4304,7 +4305,7 @@ protected void applyStartEventProperties(StartEvent se, Map<String, String> prop
     }
 
     protected void applyMessageProperties(Message msg, Map<String, String> properties) {
-        if(properties.get("name") != null) {
+        if(properties.get("name") != null && properties.get("name").length() > 0) {
```

Comment:
Agreed but not in this PR since this is 6.4.x. Let's get this change trickle down to 6.4.x through master via a separate PR specific to this refactoring.